graphql_client_cli/src/main.rs

```diff
@@ -101,13 +100,8 @@ fn introspect_schema(
     Ok(serde_json::to_writer_pretty(out, &json)?)
 }
 
-fn set_headers(authorization: Option<String>) -> Headers {
-    let mut headers = Headers::new();
-    headers.set(Accept(vec![qitem(mime::APPLICATION_JSON)]));
-
-    match authorization {
-        Some(token) => headers.set(reqwest::header::Authorization(token)),
-        None => (),
-    };
+fn construct_headers() -> HeaderMap {
+    let mut headers = HeaderMap::new();
+    headers.insert(CONTENT_TYPE, HeaderValue::from_static("application/json"));
```

Comment:
in addition to Content-Type, we should also set the [Accept](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) header (to `application/json` as well).

Reply:
@tomhoule Oops! Thank you for finding my oversight.